### PR TITLE
Events

### PR DESCRIPTION
**Pull request by alanvardy**
I realized that my posts were poorly named.

I am now using 'Events' so we can announce both meetups and anything else we plan in the future.

I removed the Meetup page as it is now redundant and added a link on the Navbar

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
+class EventsController < ApplicationController
+  before_action :set_event, only: [:show, :edit, :update, :destroy]
```
Comment (JoshMacSween) [inline app/controllers/events_controller.rb]: I'd really like to learn more about controllers, this looks great and Id like to give you more detailed comments in the future. Can we pick this apart line for line at our next meeting? :)

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+  # POST /events
+  # POST /events.json
+  def create
+    @event = Event.new(event_params)
+
+    respond_to do |format|
+      if @event.save
+        format.html { redirect_to @event, notice: 'Event was successfully created.' }
+        format.json { render :show, status: :created, location: @event }
```
Comment (JoshMacSween) [inline app/controllers/events_controller.rb]: Especially here, I'd really like to know what this json section is doing.

Comment (alanvardy) [reply app/controllers/events_controller.rb]: I don't have a super good overview... I think that if the browser requests html it returns lines 31 and if it requests JSON it returns 32. We should dig into it further together.

File: app/controllers/events_controller.rb (hunk 1 above)
Comment (alanvardy) [reply app/controllers/events_controller.rb]: I can definitely help you with this, it is under the Callbacks section of Rails Guides and we will look at it together too!

For a brief explanation: Before running any of the actions in the list (:show, :edit, :update, :destroy), it runs the set_event method down at the bottom of the file. This just dries up the code so we don't have to write set_event in every single action (a convenience method).

**Pull request merged**